vnet/spec/vnet/endpoints/1.0/networks_spec.rb

```diff
@@ -21,6 +21,31 @@ def app
   include_examples 'GET /:uuid'
   include_examples 'DELETE /:uuid'
 
+  describe 'DELETE /:uuid' do
```

Comment:
We should have this test for segments also, and for when ip_leases where the associated mac_lease or segment is deleted.